app/views/modals/loginDialog.html

```diff
@@ -12,12 +12,12 @@ <h3 ng-if="status401" translate>Authorization Required</h3>
       Please sign in with one of the following services:
     </p>
     <div class="row">
-      <div class="col-sm-5 col-sm-offset-1">
+      <div ng-if="facebookLogin" class="{{relayLogin ? 'col-sm-5 col-sm-offset-1' : 'col-sm-6 col-sm-offset-3' }}">
         <a class="btn btn-subtle btn-block" ng-href="{{apiUrl}}auth/facebook/authorization" title="Facebook"><img src="../../img/signin-facebook.png"></a>
         <a href="https://www.facebook.com/r.php" target="_blank" translate>Create a Facebook account</a>
       </div>
-      <div class="col-sm-5">
-        <a class="btn btn-subtle btn-block" ng-href="{{apiUrl}}auth/relay/login?logoutCallbackUrl={{apiUrl}}auth/relay/logout" title="Relay"><img src="../../img/signin-relay.png"></a>
+      <div ng-if="relayLogin" class="{{facebookLogin ? 'col-sm-5' : 'col-sm-6 col-sm-offset-3'}}">
```

Comment:
Are you using the correct `class` here? As in `<div ng-if="relayLogin" class="{{facebookLogin ? '` @emiliano1

Reply:
@hlbraddock Yes, the class is correct.
If the Relay button appears alone, the class will be `col-sm-6 col-sm-offset-3`.
If it appears together with the Facebook button, the class will be `col-sm-5` (as it was before).